### PR TITLE
using DataURI for widget fname call

### PR DESCRIPTION
**Pull request by davidmsibley**
requires https://github.com/UW-Madison-DoIT/uw-frame/pull/443

### CODE REVIEW

**Comment by davidmsibley** [comment]
also, it runs the first call before `widget` is populated, so you have to click update first in order to see it work.  I expect we'll iron that out soon.

**Comment by Doug-Reed** [comment]
👍

**Comment by davidmsibley** [comment]
looking at it now, if we need to call `$route.reload()` every time there's a change, I'm not convinced this is better than adding functionality to frame

File: widget-creator-war/src/main/webapp/js/override.js [hunk 1]
```diff
@@ -8,10 +8,16 @@ define(['angular'], function(angular) {
...
             'title': 'Widget Creator',
-            'fname': 'widget-creator'
+            'fname': 'widget-creator',
           },
           'SERVICE_LOC': {
-            'templates': '/json/starter-templates'
+            'templates': '/json/starter-templates',
+            'widgetApi': {
+              // For local testing, change to 'staticFeeds/'
+              'entry': 'data:application/json;base64,',
```
Comment (davidmsibley) [inline widget-creator-war/src/main/webapp/js/override.js]: probably no reason to base64 encode this

File: widget-creator-war/src/main/webapp/js/override.js [hunk 2]
```diff
@@ -8,10 +8,16 @@ define(['angular'], function(angular) {
         .constant('OVERRIDE', {
           'NAMES': {
             'title': 'Widget Creator',
-            'fname': 'widget-creator'
+            'fname': 'widget-creator',
```
Comment (vertein) [inline widget-creator-war/src/main/webapp/js/override.js]: extra comma?

Comment (davidmsibley) [inline widget-creator-war/src/main/webapp/js/override.js]: trailing comma, google style.

Comment (davidmsibley) [inline widget-creator-war/src/main/webapp/js/override.js]: Cajun Style

**Pull request closed without merging**